addons/web/static/tests/views/fields/many2many_tags_avatar_field_tests.js

```diff
@@ -154,28 +155,28 @@ QUnit.module("Fields", (hooks) => {
             target.querySelector(
                 ".o_data_row:nth-child(2) .o_field_many2many_tags_avatar img.o_m2m_avatar"
             ).dataset.src,
-            "/web/image/partner/1/avatar_128",
+            `${getOrigin()}/web/image/partner/1/avatar_128`,
             "should have correct avatar image"
         );
         assert.strictEqual(
             target.querySelector(
                 ".o_data_row:nth-child(2) .o_field_many2many_tags_avatar .o_avatar:nth-child(2) img.o_m2m_avatar"
             ).dataset.src,
-            "/web/image/partner/2/avatar_128",
+            `${getOrigin()}/web/image/partner/2/avatar_128`,
             "should have correct avatar image"
         );
         assert.strictEqual(
             target.querySelector(
                 ".o_data_row:nth-child(2) .o_field_many2many_tags_avatar .o_avatar:nth-child(3) img.o_m2m_avatar"
             ).dataset.src,
-            "/web/image/partner/4/avatar_128",
+            `${getOrigin()}/web/image/partner/4/avatar_128`,
```

Comment:
prefer full assert rather than includes (I want to see the unique key in all tests)